packages/compass-crud/src/components/readonly-filter.tsx

```diff
@@ -33,11 +32,15 @@ const QueryLabel: React.FunctionComponent<QueryLabelProps> = ({
   );
 };
 
-const textInputStyles = css({
-  input: {
-    fontFamily: fontFamilies.code,
-  },
+const readOnlyFilterStyles = css({
+  padding: spacing[2],
+  overflow: 'scroll',
   width: '100%',
+  whiteSpace: 'nowrap',
+  fontFamily: fontFamilies.code,
```

Comment:
You'd need to apply it to the code tag here because user-agent styles applied to it take precedence over parent styles, it won't inherit them by default:

![image](https://github.com/mongodb-js/compass/assets/5036933/2df850b7-d7ce-4e39-b457-b084d8f17e1c)

Reply:
I wonder .. can it be that we just need to wrap everything in a <Body> element?

Reply:
I'm not sure what you mean by that. The `code` element has default, browser-applied, font-family styles. You can't override them by applying styles to parent elements, you'd need to actually target this element with a selector to change it's styles, that's just how css works

You can also not use `code` here at all, it doesn't make any difference and doesn't add any accessibility features

Reply:
no what i mean is, shouldn't the `<Body as=div>` or some other root LG element be doing that? I would expect everything to have the right line height, base font size and font families from an element on. We could do that on our end, but if feels a bit strange

Reply:
That's not exactly how css works. There are two things to take into account here: by default if no value is set for a css property, whether or not parent will affect it will depend on whether or not property is inherited. In case of `font` is the property is inherited, yes, but another thing, as I said, `code` element has a specific style set for font-family by the browser, so whatever it inherits from the parent, is overriden by an explicit type set by the user agent

I'm not sure what you mean by font-size, I see you set it to 14px for this specific element. That overrides the 13px value inherited from a parent element that sets it to 13px value as expected for an inherited property

Reply:
I mean, we don't need anything for this PR, but I really don't think we are understanding each other, Lg has 2 base font sizes, 13 shouldn't be one of them.

My knowledge of css is a bit outdated, but I should know how a property propagates from the parent to children, that is not a recent feature. What I'm saying is that I would expect this css to be provided by LG somewhere (i thought as part of <Body>):

```
.lg-body-classname code {
   font-family: the right monospace one
}
```

which, in my book would override the default browser styles, cause the selector is more specific. The reasoning here is that having font families and sizes misaligned accidentally in a portion of the app isn't really good.

Reply:
Ah, yes, then we are talking about different things. Leafygreen Body is a paragraph and I don't think it's expected to be used the way we are using them, this is not a root component and there is no concept for something like that in lg and so I kinda get why they are not doing what you expect them to do. They do have an [InlineCode](https://www.mongodb.design/component/typography/) component exported from typography it seems

Reply:
To clarify, I think what they expect us to do is not 

```
<Body>
  <code> ...
  <p> ...
  <a> ....
</Body>
```

and expect it to provide styles for children, but just use their typography components directly where applicable

```
<div>
  <InlineCode> ...
  <Body> ...
  <Link> ...
</div>
```

Reply:
Ahh yeah, ok that's what i was looking for thanks!